users/forms.py

```diff
@@ -128,3 +128,18 @@ def is_valid(self, user):
             (user_level == User.SUB_JUDGE or user_level == User.USER):
             return True
         return False
+
+class UserForgetPasswordForm(forms.Form):
+    username = forms.CharField()
+    email = forms.EmailField()
+
+    def clean_email(self):
+        # Check that if username and email match or not
+        username = self.cleaned_data['username']
+        email = self.cleaned_data['email']
+        if username and email and User.objects.filter(username=username, email=email):
```

Comment:
I mean, should we tell the user that he/she give us an username which does not exist?